src/data-provider.js

```diff
@@ -75,7 +75,7 @@ let physicalStructProvider = ([initialNodes, initialContainers]) => {
                 let imageNameRegex = /([^/]+?)(\:([^/]+))?$/;
                 let imageNameMatches = imageNameRegex.exec(cloned.Spec.ContainerSpec.Image);
                 let tagName = imageNameMatches[3];
-                let dateStamp = dt.getDate() + "/" + (dt.getMonth() + 1) + " " + dt.getHours() + ":" + dt.getMinutes();
+                let dateStamp = (1900+dt.getYear()) + "-" + ('0' + (dt.getMonth() + 1)).slice(-2) + "-" + ('0' + dt.getDate()).slice(-2) + " " + ('0' + dt.getHours()).slice(-2) + ":" + ('0' + dt.getMinutes()).slice(-2);
```

Comment:
How about `dt.getFullYear().toString().substr(2,2)` for the year so it fits better to to the square (no need for the 2 first numbers for a long time)

Reply:
I shrunk the zoom to make a cleaner tighter picture.

When it's full, there's plenty of space.  BUT, yes, I will do `dt.getFullYear()` because I derped.

![image](https://user-images.githubusercontent.com/387006/38179115-3b9c9ddc-35ed-11e8-8ac8-f8d67e9a6cc0.png)

Reply:
👍

Reply:
you could also add that image to the readme, the one in it needs an update 😄

Reply:
@pahakalle I will add one final commit updating the README.md with a new image once we finalize all the visual bits and changes.

Do we want anything else for this PR, or shall I do that now?